Emby.Server.Implementations/Data/SqliteItemRepository.cs

```diff
@@ -6164,76 +6178,74 @@ public List<MediaAttachment> GetMediaAttachments(MediaAttachmentQuery query)
 
             if (query.Index.HasValue)
             {
-                cmdText += "AND AttachmentIndex=@AttachmentIndex";
+                cmdText += " AND AttachmentIndex=@AttachmentIndex";
             }
 
             cmdText += " order by AttachmentIndex ASC";
 
-            using (var connection = GetConnection(true))
+            var list = new List<MediaAttachment>();
+            using var connection = GetConnection(true);
```

Comment:
is this correct? I think it should be `using (var connection = GetConnection(true))`, i.e. wrapped in braces and without semicolon

Reply:
I’m 85% sure this is new C#8.0 using syntax.